students/JaNiece/life.html

```diff
@@ -19,7 +19,11 @@
   </header>
 
   <section class="now">
+<<<<<<< HEAD
```

Comment:
Same thing here (merge conflict). Probably good to do a "find" across all your files to see if there are any more instances of `<<<<<<<` hiding out.